### PR TITLE
docs(Dialog) added missing props and fixed story events

### PR DESCRIPTION
**Pull request by hobbitronics**
### Changed
- Readme index with default values for Dialog
- Dialog event names
- let prettier redo some formatting

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,11 +58,11 @@ Card (secondary, outlined, color, isClickable, noPadding)
 Checkbox (label, checked, disabled, uppercase)
 
 Datatable (label) [
-	Datatable.Data, Datatable.Data.Row (clickable), Datatable.Data.Row.Item, Datatable.Header, Datatable.Header.Item
+Datatable.Data, Datatable.Data.Row (clickable), Datatable.Data.Row.Item, Datatable.Header, Datatable.Header.Item
 ]
 
 Dialog [
-	Dialog.Alert (open, title, defaultAction), Dialog.Simple(open, title)
+Dialog.Alert (buttons: [{label: 'cancel', action: 'cancel',class: 'mdc-dialog__button'},{label: 'discard', action: 'discard',class: 'mdc-dialog__button'}], open: false, title: '', defaultAction: 'discard'), Dialog.Simple(items: [], open: false, title: '')
```
Comment (forevermatt) [inline README.md:65]: This seems to be (one of?) the only place(s) showing values for the properties. Were the values added intentionally?

Comment (hobbitronics) [reply README.md:65]: yes, I originally did not include values. If you think it is a bad idea we can remove them. I have only added them to this elements so far.

Comment (forevermatt) [reply README.md:65]: I think it's a little confusing, but perhaps that's just the formatting

File: README.md [hunk 2]
```diff
@@ -53,11 +58,11 @@ Card (secondary, outlined, color, isClickable, noPadding)
 Checkbox (label, checked, disabled, uppercase)
 
 Datatable (label) [
-	Datatable.Data, Datatable.Data.Row (clickable), Datatable.Data.Row.Item, Datatable.Header, Datatable.Header.Item
+Datatable.Data, Datatable.Data.Row (clickable), Datatable.Data.Row.Item, Datatable.Header, Datatable.Header.Item
```
Comment (forevermatt) [inline README.md:61]: Maintaining the indentation here, and perhaps putting each indented item on its own line, would probably help the readability of this.

**Review by forevermatt — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (hobbitronics) [reply README.md]: I have made it more readable now.

**Pull request merged**